### DISCUSSION TITLE
Popup displayed when saved article is read

### DISCUSSION THREAD

**Discussion by abdullahkhalidmahmood** [dailydotdev/daily#446] [Feature Requests]
If i see something interesting when I open a new tab, but i do not have time to read, i save the article.  But once I read it from saved, I would love if i get a pop up if I want to keep it in saved or remove it. 

Or maybe, I can categorise it in folders on left. Articles of blockchain go under that particular section, articles for fitness go under fitness.

**Comment by rebelchris** [comment]
Hi @abdullahkhalidmahmood (long name btw) thanks for this amazing suggestion!
I'll drop it with the team and see what they think 🥳

**Reply by idoshamun** [reply]
Let's move the discussion here:
https://github.com/dailydotdev/daily/discussions/364
